.gitattributes

```diff
@@ -1 +1,4 @@
-old_py2/* linguist-documentation
+old_py2/* linguist-documentation
+
+# Set the default behavior, in case people don't have core.autocrlf set
+*.sh text eol=lf

```

Comment:
This is needed to make sure `sh` files have `LF` endings, since by default Windows tries to "be helpful" and convert them to `CRLF`.  🙄